### DISCUSSION TITLE
Medbay can trigger event gates when they are occupied

### DISCUSSION THREAD

**Discussion by Janssendel** [tsunamayo/Starship-EVO#5258] [Suggestions]
Medbays have historically been quite a useless block, unable to use logic for further purposes. I now suggest that Medbay can trigger event gates when they are occupied, so that we can create custom medbay animations when we respawn in one.

**Comment by thriple** [comment]
That'd be really cool. Dark room lights coming on. Having the medbay rotate down, vent puffing gas, the glass window sliding up. Maybe a custom message.
